### PR TITLE
Move examples

### PR DESCRIPTION
**Pull request by TimeEngineer**

### CODE REVIEW

File: examples/ntptfs/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
-name = "ntptfs-winfsp-rs"
-version = "0.1.0-alpha"
+name = "ntptfs"
+version = "0.0.0-alpha"
```
Comment (chyyran) [inline examples/ntptfs/Cargo.toml:3]: Don’t change the version number or the package name please.

File: examples/ptfs/Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
-name = "ptfs-winfsp-rs"
```
Comment (chyyran) [inline examples/ptfs/Cargo.toml:2]: Similarly, don’t change package version or name here.

**Review by chyyran — CHANGES_REQUESTED** [review]

File: examples/ptfs/Cargo.toml (hunk 2 above)
Comment (TimeEngineer) [reply examples/ptfs/Cargo.toml]: Done

File: examples/ntptfs/Cargo.toml (hunk 1 above)
Comment (TimeEngineer) [reply examples/ntptfs/Cargo.toml]: Done

**Comment by chyyran** [comment]
I will be away from a personal Windows machine from which to test this for about 7 weeks, apologies for the delay.

**Comment by chyyran** [comment]
Superceded by #15

**Pull request closed without merging**